cpp/src/join/hash_join.cu

```diff
@@ -382,10 +390,15 @@ std::size_t hash_join::hash_join_impl::full_join_size(cudf::table_view const& pr
   // Trivial left join case - exit early
   if (!_hash_table) { return probe.num_rows(); }
 
-  auto build_table = cudf::table_device_view::create(_build, stream);
-  auto probe_table = cudf::table_device_view::create(probe, stream);
+  auto flattened_probe = structs::detail::flatten_nested_columns(
+    probe, {}, {}, structs::detail::column_nullability::FORCE);
+  auto const flattened_probe_table = std::get<0>(flattened_probe);
+
+  auto build_table             = cudf::table_device_view::create(_build, stream);
+  auto d_flattened_probe_table = cudf::table_device_view::create(flattened_probe_table, stream);
```

Comment:
Sorry I missed this. We need to rename the variables here too.

Reply:
Nice catch!  I missed it as well. 😞   Fixed.